tests/unit/src/test/scala/tests/CompilersLspSuite.scala

```diff
@@ -16,27 +16,30 @@ class CompilersLspSuite extends BaseCompletionLspSuite("compilers") {
           |package a
           |object A {
           |  // @@
-          |  def completeThisUniqueName() = 42
+          |  def completeThisName() = 42
```

Comment:
This should be reverted to the way it was before any of my changes. I change the test result, because it was also indexing vals from deps at one point. This is not true anymore.